server/monitor.go

```diff
@@ -2701,6 +2701,13 @@ func (s *Server) accountDetail(jsa *jsAccount, optStreams, optConsumers, optCfg
 		Streams: make([]StreamDetail, 0, len(jsa.streams)),
 	}
 	limits := jsa.limits
```

Comment:
As Derek mentioned previous, this could be done in one line, such as: `if limits := jsa.limits; limits != nil {`, but knowing that lookup of a nil map is valid, you could even replace this with:
```
if reserved, ok := jsa.limits[_EMPTY_]; ok {
 ...
```
However, I think that there may be case (if not always) that by default we store `defaultJSAccountTiers` for the `_EMPTY_` key, which means that MaxMemory/MaxStore are set to -1, and you take uint64() which will then translate to big number, which is likely ok, just wanted to bring this to your attention.

Reply:
updated to add test that confirms that the other account without reserved quota is actually getting the `-1` limit values.